### DISCUSSION TITLE
Add the jigsaw mesh generation process

### DISCUSSION THREAD

**Discussion by changliao1025** [changliao1025/pyflowline#225] [Ideas]
This process will be similar to the dggrid process, and it will follow @donghuix example.

**Comment by changliao1025** [comment]
Each create mesh function should have an option either to read an existing mesh or generate a new mesh.

**Reply by donghuix** [reply]
Currently, I am using ex1.py as an example to test the mesh generation process. Once we are sure about the process, we can write a ``create`` function and call it in the example.

**Comment by changliao1025** [comment]
I took a quick look at the code. I am not sure whether the current method is using the GCS WGS84 system, this means that both the jigsaw part and the dual mesh part need to use WGS84 in ALL algorithms. @donghuix

**Reply by donghuix** [reply]
I think they are using WGS84, which is from the coordinate system of the shapefile. But I agree that we should specify it in the code.
